tests/behat/features/file.feature

```diff
@@ -0,0 +1,12 @@
+@d7
+Feature: Check that FileTrait works
+
+  @api
+  Scenario: Provide files for tests(internal or external)
+    When I am logged in as a user with the "administrator" role
+    Given managed file:
```

Comment:
this test is missing the actual assertions that file objects were created.

Can you please add a test step definition to `FeatureContextD7.php`: `@Then :file_name file object exists` and add it for each file below - this is to test that file objects are created as a part of this step definition. Inside of this step - you would just be doing a file lookup by name in DB (using `file_load` etc).